package.json

```diff
@@ -7,9 +7,13 @@
   "chromium-args": "--disable-features=nw2",
   "default_locale": "en",
   "scripts": {
-    "start": "NODE_ENV=development gulp debug",
+    "start": "npx run-script-os",
```

Comment:
Pretty sure you don't need `npx` when running package from `node_modules`

```
"start": "run-script-os"
```

Reply:
Yes, seemingly it is not necessary. Thanks!